records_mover/records/types.py

```diff
@@ -116,5 +187,9 @@ class UrlDetailsEntry(TypedDict):
                                               'compression': HintCompression,
                                           },
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker